project/Dependencies.scala

```diff
@@ -1,9 +1,16 @@
 import sbt._
+import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
 
 object Dependencies {
-  private val munitVersion = "0.7.29"
-  private val zioVersion   = "2.0.1"
+  private val munitVersion  = "0.7.29"
+  private val zioVersion    = "2.0.1"
+  private val scalaJavaTime = "2.3.0"
 
-  lazy val munit = "org.scalameta" %% "munit" % munitVersion
-  lazy val zio   = "dev.zio"       %% "zio"   % zioVersion
+  lazy val munit = Def.setting("org.scalameta" %%% "munit" % munitVersion)
```

Comment:
I think it's better to define dependencies directly in `build.sbt` then use `Def.settings`